app/uk/gov/hmrc/trackingconsentfrontend/assets/src/cookieSettingsPage.ts

```diff
@@ -0,0 +1,9 @@
+const castToArray = nodeList => Array.prototype.slice.apply(nodeList)
+
+document.addEventListener('DOMContentLoaded', () => {
+  castToArray(document.querySelectorAll('[data-module="cookie-settings"]')).forEach(cookieSettingsForm => {
```

Comment:
You could use the array spread syntax here - it should be transpiled by Babel.

Reply:
i.e. [...document.querySelectorAll('[data-module="cookie-settings"]')] - I could be wrong!

Reply:
I just tried replacing `const castToArray = nodeList => Array.prototype.slice.apply(nodeList)` with `const castToArray = nodeList => [...nodeList]` and it's being simplified down to `[].concat(nodeList)` which just wraps my nodeList in an array.

Reply:
Weird, it should work.